vendor/modules.txt

```diff
@@ -460,7 +460,7 @@ github.com/prometheus/common/model
 github.com/prometheus/procfs
 github.com/prometheus/procfs/internal/fs
 github.com/prometheus/procfs/internal/util
-# github.com/rook/rook v1.9.3
+# github.com/rook/rook v1.9.9
```

Comment:
The backports for this may be difficult since #1652 wasn't backported. For backport, we will need a more restricted update than the full `make deps-update` that was run for #1652.

Reply:
right, it will give errors without the changes of #1652.